src/csv.rs

```diff
@@ -99,19 +99,25 @@ fn csv_event_columns(abi_item: &ABIItem) -> Vec<String> {
     let mut sorted_inputs = abi_item.inputs.clone();
     sorted_inputs.sort_by_key(|input| !input.indexed); // Sort by indexed field in descending order
 
-    let columns = sorted_inputs
+    let columns_prefix = vec![
+        "indexed_id".to_string(),  // the column in database created is "indexed_id", not "record_id"
+        "contract_address".to_string()
+    ];
+    let columns_suffix = vec![
```

Comment:
The columns in the header of the CSV file generated didn't line up with the rows actually generated.  For the postgres csv import it didn't matter as the import doesn't utilize the header row.   but its good to get the columns lined up.   the GCP module implementation relies on alignment.